### PR TITLE
DQM/DTMonitorModule : formatting fix for gcc 6.0 misleading-indentation warning

### PR DESCRIPTION
**Pull request by gartung**
/build/cmsbuild/jenkins-workarea/workspace/build-any-ib/w/tmp/BUILDROOT/8258e37281729e179b5af9c3b033d930/opt/cmssw/slc6_amd64_gcc600/cms/cmssw/CMSSW_8_1_X_2016-06-26-2300/src/DQM/DTMonitorModule/src/DTEfficiencyTask.cc: In member function 'void DTEfficiencyTask::fillHistos(DTLayerId, int, int, int, bool)':
  /build/cmsbuild/jenkins-workarea/workspace/build-any-ib/w/tmp/BUILDROOT/8258e37281729e179b5af9c3b033d930/opt/cmssw/slc6_amd64_gcc600/cms/cmssw/CMSSW_8_1_X_2016-06-26-2300/src/DQM/DTMonitorModule/src/DTEfficiencyTask.cc:457:2: warning: this 'if' clause does not guard... [-Wmisleading-indentation]
   if(unassHit)
  ^~
/build/cmsbuild/jenkins-workarea/workspace/build-any-ib/w/tmp/BUILDROOT/8258e37281729e179b5af9c3b033d930/opt/cmssw/slc6_amd64_gcc600/cms/cmssw/CMSSW_8_1_X_2016-06-26-2300/src/DQM/DTMonitorModule/src/DTEfficiencyTask.cc:459:4: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the 'if'
    histos[2]->Fill(missingWire);
    ^~~~~~

### CODE REVIEW

File: DQM/DTMonitorModule/src/DTEfficiencyTask.cc [hunk 1]
```diff
@@ -454,9 +454,9 @@ void DTEfficiencyTask::fillHistos(DTLayerId lId,
 				  bool unassHit) {
 
  vector<MonitorElement *> histos =  histosPerL[lId];
- if(unassHit)
+ if(unassHit) {
    histos[1]->Fill(missingWire);
-   histos[2]->Fill(missingWire);
```
Comment (gartung) [inline DQM/DTMonitorModule/src/DTEfficiencyTask.cc]: histos[2]->Fill is always called

File: DQM/DTMonitorModule/src/DTEfficiencyTask.cc [hunk 2]
```diff
@@ -456,7 +456,7 @@ void DTEfficiencyTask::fillHistos(DTLayerId lId,
  vector<MonitorElement *> histos =  histosPerL[lId];
  if(unassHit)
    histos[1]->Fill(missingWire);
-   histos[2]->Fill(missingWire);
```
Comment (gartung) [inline DQM/DTMonitorModule/src/DTEfficiencyTask.cc]: This is always called, but maybe that is what the author intended.

**Comment by dmitrijus** [comment]
please test

**Comment by davidlange6** [comment]
@gartung asks a good question - hopefully @dmitrijus is investigating - I'll merge as the change has no effect

**Pull request merged**